recipe/meta.yaml

```diff
@@ -1,34 +1,90 @@
+# Notes: this receipt is based on torchvision-feedstock: https://github.com/conda-forge/torchvision-feedstock/blob/master/recipe/meta.yaml
+
 {% set name = "torchani" %}
```

Comment:
Please remove this variable (i.e. fill it in everywhere it appears in the recipe)

Reply:
Removed

Reply:
You're going to have to replace this under package/name

Reply:
I am curious why do you ask this?

Reply:
> 
> 
> I am curious why do you ask this?

Because it is a fake degree of freedom. It's not like changing that variable is ever feasible - it's baked into the feedstock, and IMO it just serves as a stumbling block while reading (or copying the URL to calculate the hash update).

Reply:
sorry I misunderstood that, it is fixed now